WalkingChallenge/Notifications/NotificationTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NotificationTableViewCell.swift
+//  WalkingChallenge
+//
+//  Created by Husein Kareem on 4/12/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class NotificationTableViewCell: UITableViewCell {
+
+  static let CellIdentifier: String = "NotificationCell"
+
+  lazy var notificationInfoLabel: UILabel = UILabel()
+  lazy var notificationDateLabel: UILabel = UILabel()
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.backgroundColor = Style.Colors.DeepTurquoise.withAlphaComponent(0.0735)
+    self.layer.masksToBounds = false
+    self.layer.renderShadow()
+
+    self.addSubview(self.notificationInfoLabel)
+    self.addSubview(self.notificationDateLabel)
+
+    self.notificationInfoLabel.snp.makeConstraints { (make) in
+      make.top.equalTo(self).offset(10)
```

Comment:
Please use the `Style` constants for the offset (although, if there isn't one for 10 … we should see if there are other places that use that and if so create it else leave it be)

Reply:
nope, there isn't one for 10 and this is the only place i'm using it. maybe i can use 8 or 12 or i can leave it.